File: src/main/java/com/likelionknu/applyserver/auth/exception/GoogleAuthenticaionFailedException.java
package com.likelionknu.applyserver.auth.exception;

import com.likelionknu.applyserver.common.response.ErrorCode;
import com.likelionknu.applyserver.common.response.GlobalException;

public class GoogleAuthenticaionFailedException extends GlobalException {
    public GoogleAuthenticaionFailedException() {
        super(ErrorCode.GOOGLE_AUTHENTICATION_FAILED);
    }
}

File: src/main/kotlin/com/likelionknu/applyserver/auth/exception/GoogleAuthenticaionFailedException.kt
package com.likelionknu.applyserver.auth.exception

import com.likelionknu.applyserver.common.response.ErrorCode
import com.likelionknu.applyserver.common.response.GlobalException

class GoogleAuthenticaionFailedException : GlobalException(ErrorCode.GOOGLE_AUTHENTICATION_FAILED)